lib/phoenix_ecto/sql/sandbox.ex

```diff
@@ -14,9 +14,25 @@ defmodule Phoenix.Ecto.SQL.Sandbox do
       if Application.get_env(:your_app, :sql_sandbox) do
         plug Phoenix.Ecto.SQL.Sandbox
       end
-  
+
   It's important that this is at the top of `endpoint.ex`, before any other plugs.
 
+
+        plug Phoenix.Ecto.SQL.Sandbox,
+          checkout: "/sandbox/checkout"
+          checkin: "/sandbox/checkin"
```

Comment:
The current checkin configuration seems to also expect a repo? But I prefer as it is in this example and we can pass the repo as another option.